lib/src/license_detection/crc32.dart

```diff
@@ -0,0 +1,273 @@
+//CRC specification
+// Width  : 32
+// Poly   : 0x04c11db7
+// Init   : 0xffffffff
+// RefIn  : true
+// RefOut : true
+// Decimal valyue type: Unsigned
+int calculateCrc(List<int> bytes) {
```

Comment:
```suggestion
int crc32(List<int> bytes) {
```

Or maybe we can give it a better name...